lib/ui/pedals/clouds.lua

```diff
@@ -0,0 +1,98 @@
+--- CloudsPedal
+-- @classmod CloudsPedal
+
+local ControlSpec = require "controlspec"
+local UI = require "ui"
+local Pedal = include("lib/ui/pedals/pedal")
+local Controlspecs = include("lib/ui/util/controlspecs")
+
+local CloudsPedal = Pedal:new()
+-- Must match this pedal's .sc file's *id
+CloudsPedal.id = "clouds"
+
+function CloudsPedal:new(bypass_by_default)
+  local i = Pedal:new(bypass_by_default)
+  setmetatable(i, self)
+  self.__index = self
+
+  i.sections = {
+    {"Pitch/Pos", "Size/Dens"}, 
+    {"Tex/Spread", "Mode/Feedback"},
+    i:_default_section(),
+  }
+  i:_complete_initialization()
+
+  return i
+end
+
+function CloudsPedal:name(short)
+  return short and "CLDS" or "Clouds"
+end
+
+function CloudsPedal.params()
+  local id_prefix = CloudsPedal.id
+
+  local pitch_control = {
+    id = id_prefix .. "_pit",
+    name = "Pitch",
+    type = "control",
+    controlspec = ControlSpec.new(-48, 48, "lin", 1, 0, ""),
+  }
+  local pos_control = {
+    id = id_prefix .. "_pos",
+    name = "Position",
+    type = "control",
+    controlspec = ControlSpec.new(0.00, 1.00, "lin", 0.01, 0.5, ""),
+  }
+  local size_control = {
+    id = id_prefix .. "_size",
+    name = "Size",
+    type = "control",
+    controlspec = ControlSpec.new(0.00, 1.00, "lin", 0.01, 0.5, ""),
+  }
+
+  local dens_control = {
+    id = id_prefix .. "_dens",
+    name = "Density",
+    type = "control",
+    controlspec = ControlSpec.new(0.00, 1.00, "lin", 0.01, 0.5, ""),
+  }
+
+  local tex_control = {
+    id = id_prefix .. "_tex",
+    name = "Texture",
+    type = "control",
+    controlspec = ControlSpec.new(0.00, 1.00, "lin", 0.01, 0, ""),
```

Comment:
for 0-100 values sent to SC as 0-1 values with non-50/0.5 defaults, there's `Controlspecs.mix(default)`
```suggestion
    controlspec = Controlspecs.mix(0)
```